github/table_github_issue.go

```diff
@@ -52,6 +52,8 @@ func sharedIssueColumns() []*plugin.Column {
 		{Name: "assignees_total_count", Type: proto.ColumnType_INT, Transform: transform.FromField("Assignees.TotalCount", "Node.Assignees.TotalCount"), Description: "Count of assignees on the issue."},
 		{Name: "comments_total_count", Type: proto.ColumnType_INT, Transform: transform.FromField("Comments.TotalCount", "Node.Comments.TotalCount"), Description: "Count of comments on the issue."},
 		{Name: "labels_total_count", Type: proto.ColumnType_INT, Transform: transform.FromField("Labels.TotalCount", "Node.Labels.TotalCount"), Description: "Count of labels on the issue."},
+		{Name: "labels_src", Type: proto.ColumnType_JSON, Transform: transform.FromField("Labels.Nodes", "Node.Labels.Nodes"), Description: "The first 100 labels associated to the issue."},
+		{Name: "labels", Type: proto.ColumnType_JSON, Description: "A map of label names associated with this issue, in Steampipe standard format.", Transform: transform.From(getIssueLabels)},
```

Comment:
```suggestion
		{Name: "labels", Type: proto.ColumnType_JSON, Description: "A map of labels for the issue.", Transform: transform.From(getIssueLabels)},
```

I think a shorter description is OK, as "Steampipe standard format" isn't that specific. This also matches the description we use in other plugins.